config/config.go

```diff
@@ -60,6 +60,12 @@ type Config struct {
 			Avatar      string `yaml:"avatar"`
 		} `yaml:"bot"`
 
+		Limits struct {
+			MaxPuppetLimit      uint `yaml:"max_puppet_limit"`
+			MinPuppetActiveDays uint `yaml:"min_puppet_activity_days"`
+			BlockOnLimitReached bool `yaml:"block_on_limit_reached"`
```

Comment:
We also need to allow users to drop off after N days. Perhaps `max_puppet_inactivity_days`